src/services/notifications/notifications.hooks.ts

```diff
@@ -1,9 +1,32 @@
 import * as authentication from '@feathersjs/authentication';
 import { setField } from 'feathers-authentication-hooks';
+import validate from 'feathers-validate-joi';
+import {
+  createNotificationSchema,
+  getNotificationFilters,
+  notificationId,
+  updateNotificationSchema,
+} from './notifications.validations';
+import isValidId from '../../hooks/isValidId';
+import { iff, isProvider } from 'feathers-hooks-common';
+import isValidQueryParam from '../../hooks/isValidQueryParam';
+import isExistingUser from '../../hooks/isExistingUser';
+import { Hook, HookContext } from '@feathersjs/feathers';
 // Don't remove this comment. It's needed to format import lines nicely.
 
 const { authenticate } = authentication.hooks;
 
+/**
+ * Add isRead = false query param to update only not read notifications
+ * */
+function setNotificationStatus(): Hook  {
```

Comment:
This could be simplified juts to 

```
function setNotificationStatus(context: HookContext): HookContext {
  context.params.query = {
    isRead: false,
    ...context.params.query,
  };
  return context;
}
```

Reply:
changed 65367bca7e087571fc8c73a4457f1f32f0972799